Solar-Fullstack-LLM-101/05_1_ChromaDB.ipynb

```diff
@@ -181,14 +181,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/rektpunk/Library/Caches/pypoetry/virtualenvs/cookbook-pEbIoE7q-py3.12/lib/python3.12/site-packages/langchain_upstage/layout_analysis.py:135: UserWarning: UpstageLayoutAnalysisLoader is deprecated.Please use langchain_upstage.document_parse.UpstageDocumentParseLoader instead.\n",
```

Comment:
Can you remove these lines with your local directory paths?

Reply:
Sure. I removed those lines.